builder/vmware/iso/step_create_vmx.go

```diff
@@ -339,6 +341,9 @@ func (s *stepCreateVMX) Run(ctx context.Context, state multistep.StateBag) multi
 		}
 	}
 
+	// Set virtual hardware-assisted virtualization.
+	templateData.HardwareAssistedVirtualization = map[bool]string{true: "TRUE", false: "FALSE"}[config.HardwareAssistedVirtualization]
```

Comment:
This is a nice approach to get the formatted string for the boolean. But you can also rely on the template engine to do this work for you. Check out this snipped of code to see the template engine in action https://go.dev/play/p/tXPIrP88gqn

```suggestion
templateData.HardwareAssistedVirtualization = config.HardwareAssistedVirtualization
```